test-resources/infrastructure/template.yaml

```diff
@@ -32,25 +32,41 @@ Parameters:
     Default: common-cri-api
 
 Mappings:
+  KeyRotationMapping:
+    di-ipv-cri-address-api:
+      dev: true
+      build: true
+      staging: true
+      integration: false
+      production: false
+    di-ipv-cri-kbv-api:
+      dev: true
+      build: true
+      staging: true
+      integration: false
+      production: false
+    di-ipv-cri-check-hmrc-api:
+      dev: false
+      build: false
+      staging: false
+      integration: false
+      production: false
   TestHarnessUrl:
-      di-ipv-cri-check-hmrc-api:
-        localdev: review-hc.dev.account.gov.uk
-        dev: review-hc.dev.account.gov.uk
-        build: review-hc.build.account.gov.uk
-        staging: review-hc.staging.account.gov.uk
-        integration: review-hc.integration.account.gov.uk
```

Comment:
This might be a rebase issue - Suraj added the staging URLs yesterday please can we add them back in?